### ISSUE TITLE
MailChimp RSS Feed Issue

### ISSUE THREAD

**Issue by EdwardCourt** [TryGhost/Ghost#5051]
I tried adding my Ghost RSS Feed URL to MailChimp but it does not recognize it as a valid RSS Feed. I used the RSS feed from other people's older Ghost blog and it worked with no issues. Is anyone else having this problem?

**Comment by EdwardCourt** [comment]
For example if i use "https://ghost.org/rss/" for the RSS Feed URL on MailChimp it does not work. 

Error:
https://ghost.org/rss/ is not a valid RSS feed.

**Comment by letsjustfixit** [comment]
That page responds with json content: thats not valid rss all right.
However that page mostly holds static content.
What you should be looking for are relase news:
http://blog.ghost.org/rss/
or developer news:
http://dev.ghost.org/rss/

**Comment by ErisDS** [comment]
Hi @EdwardCourt, 

Firstly, as @letsjustfixit pointed out ghost.org isn't a Ghost blog, so ghost.org/rss isn't an RSS feed generated by Ghost, so it's not a good example. You've said you tried out other 'older' blogs and it worked, but you've not given us any details about your blog version, etc.

We use Mailchimp rss to email campaigns for both the blog and dev blog for Ghost, so we're pretty confident it works. This points to there being a problem either with your specific blog or perhaps with your version. Therefore there is nothing we can do without more information.

I'm going to close this issue as I don't think there's a bug in the software that needs to be fixed. The [forum](https://ghost.org/forum) is better suited for getting blog-specific support, and if you do find or have more information that demonstrates there is a bug we need to fix, let us know and we'll reopen this.

**Comment by EdwardCourt** [comment]
After further research I have narrowed down the application causing the issue. I am running Ghost 5.10 and tested the RSS feed from MailChimp using "npm -production start" to start Ghost with no problems. When I am use Nginx to proxy to my Ghost instance is what causes the RSS problem. Still investigating to identity the exact issue with Nginx that causes the error when it proxy the requests to the Ghost instance.

**Comment by acrogenesis** [comment]
@EdwardCourt were you able to fix this? I'm having this problem

**Comment by ErisDS** [comment]
The best way to figure out if a feed is valid is to paste it into here: https://validator.w3.org/feed/ If it is not, it will also tell you why. It could be due to this bug: https://github.com/dylang/node-rss/issues/49

**Comment by acrogenesis** [comment]
Thanks! That helped me solve it. Just for future reference I'll describe my error and what solved it.
dylang/node-rss#49 wasn't my bug when I validated my RSS I got an SSL error:
```
Server returned [Errno 1] _ssl.c:504: error:14077438:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert internal error
```
This is caused because w3.org dosen't support Cloudflare's SSL, it turns out mailchimp doesn't support this type SSL either. So I created a page rule on Cloudflare to serve the /rss path always without SSL.
